squashfs/squashfs.go

```diff
@@ -176,21 +185,21 @@ func ExtractSingleSquash(squashFile string, extractDir string, storageType strin
 		cmd.Stdin = nil
 		return cmd.Run()
 		return cmd.Start()
-	} else if p := which("squashfuse"); p != "" {
+	} else if sqfuse := findSqfusePath(); sqfuse != "" {
 		// given extractDir of path/to/some/dir[/], log to path/to/some/.dir-squashfs.log
 		extractDir := strings.TrimSuffix(extractDir, "/")
 
 		var cmdOut io.Writer
 		logf := path.Join(path.Dir(extractDir), "."+path.Base(extractDir)+"-squashfuse.log")
 		if cmdOut, err = os.OpenFile(logf, os.O_RDWR|os.O_TRUNC|os.O_CREATE, 0644); err != nil {
-			log.Infof("Failed to open %s for write: %v", p, err)
+			log.Infof("Failed to open %s for write: %v", logf, err)
```

Comment:
the logf fix is needed regardless, independent of this commit, right?

Reply:
Yes